.github/workflows/tests_and_reports.yml

```diff
@@ -148,17 +158,19 @@ jobs:
           cache: npm
 
       - name: Install
+        run: npm i -g lighthouse lighthouse-batch
+
+      - name: Construct URLs to test
         run: |
-          npm ci
-          npm i -g lighthouse lighthouse-batch
+          while IFS="" read -r path || [ -n "$path" ]
+          do
+            echo "${{ steps.vercel_preview_url.outputs.url }}${path}" >> .lighthouse/urls.txt
+          done < .lighthouse/paths.txt
 
       - name: Generate Lighthouse reports
         run: |
-          npm run build
-          npm run preview &
-          sleep 5
-          lighthouse-batch --use-global --html --out lighthouse-reports/mobile --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\""
-          lighthouse-batch --use-global --html --out lighthouse-reports/desktop --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\" --preset=desktop"
+          lighthouse-batch --use-global --html --out lighthouse-reports/mobile --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\" --throttling.cpuSlowdownMultiplier=8"
+          lighthouse-batch --use-global --html --out lighthouse-reports/desktop --file .lighthouse/urls.txt --params "--only-categories=\"performance,accessibility,best-practices,seo\" --throttling.cpuSlowdownMultiplier=8 --preset=desktop"
```

Comment:
Do these throttling settings come from somewhere specific? Are these to more closely match user devices? I was under the impression that GitHub Actions runners are already pretty slow.

Reply:
This was suggested in the initial set of reports generated, since the GitHub Actions runner was struggling to keep up with the base slowdown modifier of `4`. I'll add a comment to the workflow to note this.